src/toxcore/dht/kbucket.rs

```diff
@@ -447,15 +447,15 @@ impl Kbucket {
 
     It should not contain LAN ip node.
     */
-    pub fn get_closest(&self, pk: &PublicKey) -> Vec<PackedNode> {
+    pub fn get_closest(&self, pk: &PublicKey, from_global: bool) -> Vec<PackedNode> {
```

Comment:
rename `from_global` to `only_global_ip` and add docs to the function.

`It should not contain LAN ip node.` is wrong

Reply:
done